apps/docs/content/guides/platform/performance.mdx

```diff
@@ -175,21 +175,7 @@ You can pair the information available from `pg_stat_statements` with the detail
 
 ## Optimizing the number of connections
 
-By default, the number of connections allowed to Postgres and Supavisor is configured based on the resources available to the database.
-
-| Compute Add-on | Postgresql connections | Supavisor connections |
-| -------------- | ---------------------- | --------------------- |
-| - (Free plan)  | 30                     | 200                   |
-| Micro          | 60                     | 200                   |
-| Small          | 90                     | 200                   |
-| Medium         | 120                    | 200                   |
-| Large          | 160                    | 300                   |
-| XL             | 240                    | 700                   |
-| 2XL            | 380                    | 1500                  |
-| 4XL            | 480                    | 3000                  |
-| 8XL            | 490                    | 6000                  |
-| 12XL           | 500                    | 9000                  |
-| 16XL           | 500                    | 12000                 |
+By default, the number of connections allowed to Postgres and Supavisor is configured based on the resources available to the database. You can navigate to [Compute Add-ons](/docs/guides/platform/compute-add-ons) to check your database's direct and pooler connections.
```

Comment:
@charislam this is necessary since there's limits in too many places but i'm conflicted on the wording here. I want to convey that customers can check the default direct and pooler connections of their database instance. lmk your thoughts.

Reply:
This seems to preserve the meaning of the original:

```suggestion
The number of connections allowed to Postgres and Supavisor is based on your compute size. See the default connection numbers in the [Compute Add-ons](/docs/guides/platform/compute-add-ons) section.
```

Though actually I'm a bit confused. This paragraph suggests there are hard limits, but your note and the link suggest these are just recommendations. _Is_ there a hard limit?

If it's not a hard limit, I wouldn't say `allowed`, I would say the `default connection limit for Postgres ...`

Reply:
@charislam good feedback, thanks! I updated accordingly to:

```The default connection limits for Postgres and Supavisor is based on your compute size. See the default connection numbers in the [Compute Add-ons](/docs/guides/platform/compute-add-ons) section.```